CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Views/KitchenLineCardsViews/ItemsInProgressCardView.xaml.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CheckerUI.Helpers.Order;
+using CheckerUI.ViewModels;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace CheckerUI.Views.KitchenLineCardsViews
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class ItemsInProgressCardView : ContentView
+    {
+        public BaseLineViewModel ViewModel { get; set; } = new BaseLineViewModel();
+        public ItemsInProgressCardView()
+        {
+            InitializeComponent();
+        }
+        
+        private Frame m_LastFrameTapped;
+        private void TapGestureRecognizer_OnTapped(object sender, EventArgs e)
+        {
+            var stackLayout = sender as StackLayout;
+            m_InProgressCollection.SelectedItem = stackLayout.BindingContext;
+            var card = stackLayout.LogicalChildren[0] as KitchenOrderItemInProgressCardView;
+
+
+            var frame = card.Children[0] as Frame;
+          
+            if ( m_LastFrameTapped != null)
+            {
+                m_LastFrameTapped.BackgroundColor = Color.White;
+            }
+
+            if (m_LastFrameTapped != frame)
+            {
+                m_LastFrameTapped = frame;
+                m_LastFrameTapped.BackgroundColor = Color.BurlyWood;
+            }
```

Comment:
These colors, as most colors in use in this project, can be defined on top (in a designated Theme file! ) as static fields 
```
 static Color s_UntappedItemFrameColor = Color.White;
 static Color s_TappedItemFrameColor = Color.BurlyWood;
```

that way they can be easily accessible and changable in case of theme change.